test/cmdlineTests/require_with_error_ir/input.sol

```diff
@@ -3,15 +3,12 @@ pragma solidity >=0.0;
 
 error CustomError(uint256, string);
 
-contract C
-{
-    function f(bool condition) external pure
-    {
+contract C {
+    function f(bool condition) external pure {
```

Comment:
I think you did the wrong file :) This one was not added in this PR. Not that it matters much though :)

Reply:
Changed both of them.